### DISCUSSION TITLE
ignore  this, i was testing this for my new project

### DISCUSSION THREAD

**Discussion by PrayagN** [community/community#67941] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

response should containe proper status code

**Comment by PrayagN** [comment]
yes you should want to pass staus code with each response

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
